recipes/wqio/meta.yaml

```diff
@@ -0,0 +1,99 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
+# If your package is python based, we recommend using Grayskull to generate it instead:
+# https://github.com/conda-incubator/grayskull
+
+# Jinja variables help maintain the recipe as you'll update the version only here.
+# Using the name variable with the URL in line 16 is convenient
+# when copying and pasting from another recipe, but not really needed.
+{% set name = "wqio" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://files.pythonhosted.org/packages/43/1d/4e526babc0391529e10f63d27800f18982f501e32faf1de15c25d6877dd9/wqio-0.6.0.tar.gz
```

Comment:
Can you try generating this recipe with grayskull? It will fix things like the url templating here.

Reply:
If you manage work without adding new packages to your workflow you are a winner! Don't be sorry. The less packages the better ;-p